### PR TITLE
Update TZDB to 2022f

### LINKED ISSUE #115: tzdb 2022f
This was recently released (https://data.iana.org/time-zones/tzdb/NEWS) and contains at least one important change to time in Mexico effective Oct 30th as well as some consolidation.

### PR DESCRIPTION
**Pull request by ianthetechie**
Closes #115.

### CODE REVIEW

**Comment by djc** [comment]
Thanks, this is good stuff! I think we'll want to call this 0.8 since removing timezones makes this semver-incompatible, I think. Also would be helpful to rebase this on current master so the merge commit goes way.

**Comment by ianthetechie** [comment]
Good call. I'll do both of those real quick.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
 *.swp
+.idea/
```
Comment (LucioFranco) [inline .gitignore:4]: I think its a better idea to include this as a global gitignore item for your computer then add it to the project.

Comment (LucioFranco) [inline .gitignore:4]:
```suggestion

```

Comment (ianthetechie) [reply .gitignore]: I actually don’t necessarily always avoid committing it but yeah I can understand not wanting it. I’ve reverted that change.

**Comment by djc** [comment]
Published to crates.io.

**Pull request merged**